staging/src/k8s.io/kubectl/pkg/generated/bindata.go

```diff
@@ -1,4 +1,4 @@
-// Package generated Code generated by go-bindata. (@generated) DO NOT EDIT.
+// Code generated by go-bindata.
```

Comment:
This file showed up when I ran "make update". Unsure if I should be including it here, it seems unrelated

Reply:
apparently I should not include it